services/QuillLMS/app/workers/set_impact_metrics_worker.rb

```diff
@@ -7,20 +7,19 @@ class SetImpactMetricsWorker
   SENTENCES_PER_ACTIVITY_SESSION = 10
 
   def perform
-    finished_activity_sessions_count = ImpactMetrics::ActivitiesAllTimeQuery.run[0]["count"]
-    active_students_count = ImpactMetrics::ActiveStudentsAllTimeQuery.run[0]["count"]
+    finished_activity_sessions_count = ImpactMetrics::ActivitiesAllTimeQuery.run[0][:count]
+    active_students_count = ImpactMetrics::ActiveStudentsAllTimeQuery.run[0][:count]
 
     teachers = ImpactMetrics::ActiveTeachersAllTimeQuery.run
-    teacher_ids = teachers.to_a.map {|teacher| teacher["id"]}
 
-    schools = ImpactMetrics::SchoolsContainingCertainTeachersQuery.run(teacher_ids: teacher_ids)
-    low_income_schools = schools.select { |school| (school["free_lunches"] || 0) > FREE_LUNCH_MINIMUM}
+    schools = ImpactMetrics::SchoolsContainingCertainTeachersQuery.run
```

Comment:
Probably out of scope for this fix, but we could probably do the same thing here, i.e. make this a count query with an option for free lunch minimum.